deploy/buildtools/bitbuilder.py

```diff
@@ -201,8 +194,6 @@ def on_build_failure():
 
             message_body = "Your FPGA build failed for triplet: " + self.build_config.get_chisel_triplet()
 
-            send_firesim_notification(message_title, message_body)
```

Comment:
I don't think this was supposed to be deleted?

Reply:
Ok. I'll update this after the 1st set of CI runs.